string position
string orientation
string relative_pose
string joint_angles
string endpoint_pose

